mappings/net/minecraft/util/math/Weighting.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_6007 net/minecraft/util/math/Weighting
+	FIELD field_29927 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_29928 DEFAULT Lnet/minecraft/class_6007;
+	FIELD field_29929 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_29930 weight I
+	METHOD <init> (I)V
+		ARG 1 weight
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
+	METHOD method_34976 getWeight ()I
```

Comment:
Maybe `getValue()`, like how java annotations have `value()` by default. Also looking at the codec, it is just an int without any key, so `value` makes sense.